### PR TITLE
WF-64779-Fix the formatting issues in Presentation UG

### PR DESCRIPTION
**Pull request by chrispojunior**
<table style="width: 521.306px;">
<tbody>
<tr>
<td style="width: 156px;">Task Link</td>
<td style="width: 349.306px;">https://syncfusion.atlassian.net/browse/WF-64779</td>
</tr>
<tr>
<td style="width: 156px;">Description</td>
<td style="width: 349.306px;">Resolved formatting issues in Presentation UG</td>
</tr>
<tr>
<td style="width: 156px;">Solution</td>
<td style="width: 349.306px;">Resolve formatting issues charts, smartArt, connectors, header footers and notes pages of PPT</td>
</tr>
<tr>
<td style="width: 156px;">Content review confirmation</td>
<td style="width: 349.306px;">Yes</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: File-Formats/Presentation/Working-with-sections.md [hunk 1]
```diff
@@ -423,104 +312,72 @@ You can move the sections within a PowerPoint presentation. The following code e
...
 'Loads a PowerPoint presentation
-Dim pptxDoc As IPresentation = Presentation.Open("PPTXWithSections.PPTX")
-
-'Moves the second section to third position within the PowerPoint presentation.
-pptxDoc.Sections(2).Move(3)
-
-'Saves the PowerPoint presentation
-pptxDoc.Save("Sections.PPTX")
-
+Dim pptxDoc As IPresentatio
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-sections.md:325]: You have removed this VB code snippet. Please add it.

File: File-Formats/Presentation/Working-with-sections.md [hunk 2]
```diff
@@ -656,145 +480,102 @@ The following code example demonstrates how to clone the slide collection of a s
...
-
 'Creates a destination PowerPoint presentation instance. Existing presentations can also be used here.
 pptxDoc = Presentation.Create()
-
 'Iterates the cloned slides and adds the slides to the destination presentation
-For Each slide As ISlide In slides
-
-pptxDoc.Slides.Add(slide)
-
+    For Each slide As ISlide In slides
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-sections.md:504]: Please remove empty space before this line.

File: File-Formats/Presentation/Working-with-sections.md [hunk 3]
```diff
@@ -656,145 +480,102 @@ The following code example demonstrates how to clone the slide collection of a s
...
 pptxDoc = Presentation.Create();
-
 //Iterates the cloned slides and adds the slides to the destination presentation
 foreach (ISlide slide in slides)
-
-pptxDoc.Slides.Add(slide);
-
-//Save the PowerPoint Presentation as stream
+    pptxDoc.Slides.Add(slide);
+    //Save the PowerPoint Presentation as stream
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-sections.md:549]: Please remove empty space before this line.

File: File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md [hunk 4]
```diff
@@ -179,210 +122,136 @@ You can also add a connector to a source shape without any destination shapes. T
...
+    //Add a rectangle shape on the slide
+    IShape rectangle = slide.Shapes.AddShape(AutoShapeType.Rectangle, 420, 250, 100, 100);
+    //Add connector with specified bounds
+    IConnector connector = slide.Shapes.AddConnector(ConnectorType.Straight, 0, 0, 470, 150);
+    //Connect the beginning point of the connector with rectangle shape
+    connector.BeginConnect(rectangle, 0);
+    //Set the beginning cap of the connector as arrow
    connector.LineFormat.BeginArrowheadStyle = ArrowheadStyle.Arrow;
-   
    //Change the connector color
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md:244]: Alignment is not proper in this code snippet.

File: File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md [hunk 5]
```diff
@@ -399,250 +268,161 @@ The following code example demonstrates how to edit an existing connector in a P
...
+    //Set the line format for the connector
+    connector.LineFormat.DashStyle = LineDashStyle.DashDotDot;
+    //Disconnect the end connection of the connector if end point get connected
+    if (connector.EndConnectedShape != null)
+        connector.EndDisconnect();
+    //Insert a triangle shape into slide
+    IShape triangle = slide.Shapes.AddShape(AutoShapeType.IsoscelesTriangle, 600, 500, 150, 150);
+    //Declare the end connection site index
+    int connectionSiteIndex = 4;
    //Reconnect the end point of connector with triangle shape if its connection site count is greater than 4
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md:417]: Alignment is not proper in this code snippet.

File: File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md [hunk 6]
```diff
@@ -653,192 +433,124 @@ The following code example demonstrates how to update a connector’s position w
...
     IConnector connector = slide.Shapes[2] as IConnector;
-
     //Change the X and Y position of the rectangle
     rectangle.Left = 600;
     rectangle.Top = 200;
-
     //Update the connector to connect with previously updated shape
     connector.Update();
-
    //The operation in Save under Xamarin varies between Windows Phone, Android and iOS platforms. Please refer presentation/xamarin section for respective code samples.
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md:548]: Alignment is not proper in this line.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

File: File-Formats/Presentation/Working-with-Charts.md [hunk 7]
```diff
@@ -1239,459 +1010,321 @@ The appearance of a chart can be modified according to the convenience and usage
...
 //Hides the category labels
 chart.CategoryLabelLevel = OfficeCategoriesLabelLevel.CategoriesLabelLevelNone;
-
 //Shows Data Table.
 chart.HasDataTable = true;
-
 //Formats Chart Area.
 IOfficeChartFrameFormat chartArea = chart.ChartArea;
-
 //Chart Area Border Settings
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-Charts.md:1222]: Remove this empty line.

File: File-Formats/Presentation/Working-with-Charts.md [hunk 8]
```diff
@@ -1239,459 +1010,321 @@ The appearance of a chart can be modified according to the convenience and usage
...
 //Changes the series name of first chart series
 chart.Series[0].Name = "Modified series name";
-
 //Hides the category labels
 chart.CategoryLabelLevel = OfficeCategoriesLabelLevel.CategoriesLabelLevelNone;
-
 //Formats Chart Area.
 IOfficeChartFrameFormat chartArea = chart.ChartArea;
-
 //Chart Area Border Settings
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-Charts.md:1284]: Remove this empty line.

File: File-Formats/Presentation/Working-with-Charts.md [hunk 9]
```diff
@@ -2971,119 +2225,96 @@ The following code example demonstrates creating a Scatter chart.
...
-chart.ChartData.SetValue(4, 1, 10);                  
-chart.ChartData.SetValue(2, 2, 10);  
-chart.ChartData.SetValue(3, 2, 5);  
-chart.ChartData.SetValue(4, 2, 1);                  
+chart.ChartData.SetValue(4, 1, 10);
+chart.ChartData.SetValue(2, 2, 10);
+chart.ChartData.SetValue(3, 2, 5);
+chart.ChartData.SetValue(4, 2, 1);
+
+//Apply chart elements
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-Charts.md:2251]: Remove this empty line.

File: File-Formats/Presentation/Working-with-Charts.md [hunk 10]
```diff
@@ -2971,119 +2225,96 @@ The following code example demonstrates creating a Scatter chart.
...
-serie.DataPoints.DefaultDataPoint.DataLabels.IsCategoryName = true 
+chart.ChartData.SetValue(2, 1, 1)
+chart.ChartData.SetValue(3, 1, 5)
+chart.ChartData.SetValue(4, 1, 10)
+chart.ChartData.SetValue(2, 2, 10)
+chart.ChartData.SetValue(3, 2, 5)
+chart.ChartData.SetValue(4, 2, 1)
 
+'Apply chart elements
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-Charts.md:2290]: Remove this empty line.

File: File-Formats/Presentation/Working-with-Charts.md [hunk 11]
```diff
@@ -3094,137 +2325,112 @@ chart.ChartData.SetValue(2, 2, 10);
 chart.ChartData.SetValue(3, 2, 5);  
 chart.ChartData.SetValue(4, 2, 1);                  
 
-//Apply chart elements  
+//Apply chart elements
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-Charts.md:2329]: Remove this empty line.

File: File-Formats/Presentation/Working-with-Charts.md [hunk 12]
```diff
@@ -3094,137 +2325,112 @@ chart.ChartData.SetValue(2, 2, 10);
...
+chart.ChartData.SetValue(1, 2, "Y-Axis");
+chart.ChartData.SetValue(2, 1, 1);
+chart.ChartData.SetValue(3, 1, 5);
+chart.ChartData.SetValue(4, 1, 10);
+chart.ChartData.SetValue(2, 2, 10);
+chart.ChartData.SetValue(3, 2, 5);
+chart.ChartData.SetValue(4, 2, 1);
+
+//Apply chart elements
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-Charts.md:2373]: Remove this empty line.

File: File-Formats/Presentation/Working-with-Charts.md [hunk 13]
```diff
@@ -3094,137 +2325,112 @@ chart.ChartData.SetValue(2, 2, 10);
...
```
Comment (Suriya-Balamurugan) [inline File-Formats/Presentation/Working-with-Charts.md:2413]: Remove this empty line.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/Working-with-Charts.md (hunk 13 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-Charts.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-Charts.md (hunk 12 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-Charts.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-Charts.md (hunk 11 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-Charts.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-Charts.md (hunk 10 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-Charts.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-Charts.md (hunk 9 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-Charts.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-Charts.md (hunk 8 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-Charts.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-Charts.md (hunk 7 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-Charts.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md (hunk 6 above)
Comment (sharonaumao) [reply File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md (hunk 5 above)
Comment (sharonaumao) [reply File-Formats/Presentation/create-edit-connectors-in-powerpoint-slides-cs-vb-net.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-sections.md (hunk 3 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-sections.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-sections.md (hunk 2 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-sections.md]: @Suriya-Balamurugan -Done

File: File-Formats/Presentation/Working-with-sections.md (hunk 1 above)
Comment (sharonaumao) [reply File-Formats/Presentation/Working-with-sections.md]: @Suriya-Balamurugan -Done

**Pull request merged**